service-model-generator/modules/service-model-generator-ls-extension/src/main/java/io/ballerina/servicemodelgenerator/extension/ServiceModelGeneratorService.java

```diff
@@ -349,7 +353,7 @@ public CompletableFuture<ServiceModelResponse> getServiceModel(ServiceModelReque
                 }
                 Set<String> listenersList = ListenerUtil.getCompatibleListeners(request.moduleName(), semanticModel,
                         project);
-                serviceModel.getListener().setItems(listenersList.stream().toList());
+                serviceModel.getListener().setItems(listenersList.stream().map(l -> (Object) l).toList());
```

Comment:
Value-> setItems(List<Object> items) gets List of Objects, hence do we need to cast this to Object explicitly?

Reply:
yes, we need to otherwise there will be compile time error